src/components/RegisterForm.tsx

```diff
@@ -28,32 +32,18 @@ function RegisterForm(props: RegisterFormProps) {
     actions.setSubmitting(false);
     setLoading(true);
     try {
-      const data = {
+      const data: Attendee = {
+        eventID: id,
         ...values,
       };
-
-      //await registerEvent(data,id);
-      toast.success("You have registered successfully!", {
-        position: "top-right",
-        autoClose: 5000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: true,
-        draggable: true,
-        theme: "colored",
-      });
-    } catch {
-      toast.error(
-        "There was an error with your registration, please try again later",
-        {
-          position: "top-right",
-          autoClose: 5000,
-          hideProgressBar: false,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          theme: "colored",
-        }
+      console.log(data);
+      await addAttendee(data);
+      showToast("success", "You have registered successfully!");
```

Comment:
Create a `constants` file and add values that do not change to it. For example the types of the toasts should be added within `const TOAST_TYPES = { ... }`.